src/Controls/src/Core/Shell/Shell.cs

```diff
@@ -948,95 +1267,131 @@ public ShellItem CurrentItem
+		/// <summary>
+		/// Gets or sets the background color of the Shell Flyout.
+		/// </summary>
 		public Color FlyoutBackgroundColor
 		{
 			get => (Color)GetValue(FlyoutBackgroundColorProperty);
 			set => SetValue(FlyoutBackgroundColorProperty, value);
 		}
 
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='FlyoutBackground']/Docs/*" />
+		/// <summary>
+		/// Gets or sets the background color of the Shell Flyout.
+		/// </summary>
 		public Brush FlyoutBackground
 		{
 			get => (Brush)GetValue(FlyoutBackgroundProperty);
 			set => SetValue(FlyoutBackgroundProperty, value);
 		}
 
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='FlyoutBackdrop']/Docs/*" />
+		/// <summary>
+		/// Gets or sets the backdrop of the flyout, which is the appearance of the flyout overlay.
+		/// </summary>
 		public Brush FlyoutBackdrop
 		{
 			get => (Brush)GetValue(FlyoutBackdropProperty);
 			set => SetValue(FlyoutBackdropProperty, value);
 		}
 
+		/// <summary>
+		/// Gets or sets the width of the flyout.
+		/// </summary>
 		public double FlyoutWidth
 		{
 			get => (double)GetValue(FlyoutWidthProperty);
 			set => SetValue(FlyoutWidthProperty, value);
 		}
 
+		/// <summary>
+		/// Gets or sets the height of the flyout.
+		/// </summary>
 		public double FlyoutHeight
 		{
 			get => (double)GetValue(FlyoutHeightProperty);
 			set => SetValue(FlyoutHeightProperty, value);
 		}
 
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='FlyoutBehavior']/Docs/*" />
+		/// <summary>
+		/// Gets or sets the behavior to open the flyout.
+		/// </summary>
 		public FlyoutBehavior FlyoutBehavior
 		{
 			get => (FlyoutBehavior)GetValue(FlyoutBehaviorProperty);
 			set => SetValue(FlyoutBehaviorProperty, value);
 		}
 
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='FlyoutHeader']/Docs/*" />
+		/// <summary>
+		/// Gets or sets the View that define the appearance of the flyout header.
+		/// The flyout header is the content that optionally appears at the top of the flyout.
+		/// </summary>
 		public object FlyoutHeader
 		{
 			get => GetValue(FlyoutHeaderProperty);
 			set => SetValue(FlyoutHeaderProperty, value);
 		}
 
+		/// <summary>
+		/// Gets or sets the View that define the appearance of the flyout footer.
+		/// The flyout footer is the content that optionally appears at the bottom of the flyout.
+		/// </summary>
 		public object FlyoutFooter
 		{
 			get => GetValue(FlyoutFooterProperty);
 			set => SetValue(FlyoutFooterProperty, value);
 		}
 
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='FlyoutHeaderBehavior']/Docs/*" />
+		/// <summary>
+		/// Setting the FlyoutHeaderBehavior to CollapseOnScroll collapses the flyout as scrolling occurs. 
```

Comment:
I think that this property should conform to the documentation style simply mention that it  `Gets or sets the header behavior for the flyout` and move this docs to the  `FlyoutHeaderBehavior` class/enum